MSAL/src/native_auth/controllers/factories/MSALNativeAuthControllerFactory.swift

```diff
@@ -36,19 +36,19 @@ final class MSALNativeAuthControllerFactory: MSALNativeAuthControllerBuildable {
         self.config = config
     }
 
-    func makeSignUpController() -> MSALNativeAuthSignUpControlling {
-        return MSALNativeAuthSignUpController(config: config)
+    func makeSignUpController(cacheAccessor: MSALNativeAuthCacheInterface) -> MSALNativeAuthSignUpControlling {
```

Comment:
Do you think it would make sense to make the `cacheAccessor` part of the `MSALNativeAuthConfiguration`?

Reply:
You're right, thanks for the explanation